cli/container.go

```diff
@@ -256,32 +264,41 @@ func parseRestartPolicy(restartPolicy string) (*types.RestartPolicy, error) {
 	return policy, nil
 }
 
-func parseNetwork(network string) (string, string, error) {
+// network format as below:
+// [network]:[ip_address], such as: mynetwork:172.17.0.2 or mynetwork(ip alloc by ipam) or 172.17.0.2(default network is bridge)
+// [network_mode]:[parameter], such as: host(use host network) or container:containerID(use exist container network)
+// [network_mode]:[parameter]:mode, such as: mynetwork:172.17.0.2:mode(if the container has multi-networks, the network is the default network mode)
```

Comment:
```if the container has multi-networks, the network is the default network mode```

Actually if the container is in "none" or "host" network mode, then it can't join other networks.

If the container is in "container" mode, then sandbox which both containers are sharing can join other networks.

So if the container is in some modes (none, host, container), we can't create it with multiple networks. Then the third network format is redundant.

Reply:
yes, you are right, now it has just completed the basic functions for none and host mode simply. More features need to support and its conflicts.